content/release-10-18-0/devices-10-18-0.md

```diff
@@ -198,6 +216,55 @@ Build version</th>
 <td>Core</td>
 </tr>
 
+<tr>
+<td>LWM2M</td>
+<td>Custom actions were not always executed if a LWM2M device used a SEND operation to report the whole LWM2M object data. This issue is now resolved and the custom actions are always executed for the resource instances for which they are defined.</td>
+<td>DM-2188</td>
+<td>10.18.0.57</td>
+<td>Messaging Service</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>Bulk device registrations and other operations being executed on the LWM2M connector device now show the status FAILED if a problem occurs. Prior to this change, partial failures were reported as SUCCESSFUL.</td>
+<td>DM-1951</td>
+<td>10.18.0.54</td>
+<td>Core</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>In certain cases, the registration couldn't be associated to a LWM2M 1.1 SEND request, leading to a 4.04 CoAP response. This is now fixed.</td>
+<td>DM-2270</td>
+<td>10.18.0.47</td>
+<td>Core</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>The LWM2M agent now ignores trailing commas at the end of object links in the registration request of a LWM2M client.</td>
+<td>DM-2342</td>
+<td>10.18.0.46</td>
+<td>Core</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>The LWM2M agent is now able to properly convert the timestamps from the SenML data reported by the LWM2M client to a platform compatible date-time format for performing respective resource actions.</td>
+<td>DM-2150</td>
+<td>10.18.0.43</td>
+<td>Core</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>Multi-line LWM2M post-operations were not executed right after the LWM2M device's new registration when realtime was disabled for the tenant who owns the device. As a result the device might not receive these operations until the next LWM2M device's registration update. This issue is now resolved and LWM2M post-operations are executed right after the LWM2M device's new registration, no matter whether realtime is enabled or not for this kind of devices.</td>
```

Comment:
not sure about the post-operation (i recall that i commented on this in another PR yesterday)